tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
     "esModuleInterop": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
We shouldn't be targetting `commonjs`, Pika does that for us.

If you use the syntax i commented above, then you don't need to use the TS `import = require()`